include/zephyr/arch/x86/x86_acpi_osal.h

```diff
@@ -26,4 +26,13 @@ static inline void *acpi_rsdp_get(void)
 	return bios_acpi_rsdp_get();
 }
 #endif /* CONFIG_X86_EFI */
+
+static inline uint64_t acpi_timer_get(void)
```

Comment:
Should we require ACPI timer for acpica ?
The other question, @jhedberg will it work with qemu?

Reply:
If we need to use HPET then we cannot use system timer API and hence timer api should be a platform specific custom implementation which can be called in early init time. For QEMU etc. it is still HPET and HPET is required in case of some use case (such as power management etc.)